poll.go

```diff
@@ -15,16 +15,18 @@ import (
 var STATUSES = [5]string{"UNKNOWN", "DEPLOYED", "DELETED", "SUPERSEDED", "FAILED"}
```

Comment:
You could also change this variable name

Reply:
Yes, and I would call it **states**.